torch/csrc/jit/python/init.cpp

```diff
@@ -1490,42 +1490,32 @@ void initJITBindings(PyObject* module) {
           [](Argument& self) -> py::bool_ {
             return self.default_value().has_value();
           })
+      .def_property_readonly(
+          "alias_info", [](Argument& self) { return self.alias_info(); })
       .def_property_readonly(
           "is_out", [](Argument& self) { return self.is_out(); })
+      .def_property_readonly("kwarg_only", [](Argument& self) -> bool {
+        return self.kwarg_only();
+      });
+  py::class_<AliasInfo>(m, "AliasInfo")
```

Comment:
You can probably make this "_AliasInfo" and then you won't need to add it into the torch_C_allowlist_superset